src/views/tutorial-view/components/featured-in-collections/featured-in-collections.module.css

```diff
@@ -0,0 +1,44 @@
+.heading {
+  margin: 0;
+  padding: 0;
+  composes: hds-typography-display-200 from global;
+  font-weight: var(--token-typography-font-weight-semibold);
+  color: var(--token-color-foreground-strong);
+}
+
+.listRoot {
+  align-items: stretch;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 24px;
+  list-style: none;
+
+  /**
+   * Note: !important is necessary here as we're within a content container
+   * that uses nested generic selectors to style <ul />.
+   * TODO: convert the nested generic selectors to MDX components.
+   * Asana task:
+   * https://app.asana.com/0/1201987349274776/1201653123138729/f
```

Comment:
Going to try to resolve this issue next because it keeps coming up. I think there's two independent yet related things I'd like to tackle:
1. Move the content container out of `SidebarSidecarLayout`.
    - Proposed approach: Views like `tutorial-view` & `docs-view` will instead consume the content container, so that it can wrap the MDXContent part of things only (rather than the entire content area). I think Kendall already proposed something similar (or identical!) to this
    - No matter how well-encapsulated our content styles are, I think we want to do this. And seems relatively low-effort now that docs-view is better abstracted thanks to Ashlee's work on consolidating product-specific `additionalComponents`. So I'm gonna try to tackle this first
2. Do the MDX custom component thing for `<ul />` & `<ol />`.
    - Once we do the above container refactor, this will be less "necessary", but I'm hoping to have bandwidth to try it out anyways (eg if we add a similar "list of linked cards" or really "list of anything" custom component for use in the content area in the future, then it will come up again).

Reply:
I vote #1! Seems like the most straightforward way to address this problem and it also doesn't prevent us from pursuing #2 as we need it.

Reply:
I don't think it should block this PR, we can follow up and remove the `!important`s when we make that change.

Reply:
If you do 2 first, will 1 still be needed?

Reply:
☝️ great Q - i think either 2 or 1 will make the other much less necessary.

But I also think it's probably ideal if we do both? I guess if our styles are really well encapsulated then there's technically no harm in not doing `1` - I guess sibling selectors could still apply unexpectedly though? 🤔